app/controllers/addOrderProductCtrl.js

```diff
@@ -0,0 +1,32 @@
+'use strict'; 
+const prompt = require('prompt');
+
+module.exports.promptAvailableProducts = (products) => {
+  let list = "";
+  //build a list of products with quantity > 0
+  let availableProducts = products.filter(prod => prod.quantity > 0);
```

Comment:
Could change some of these lets to consts.

Reply:
nicee i'll change those before merge